src/lib/components/ScrollView/README.mdx

```diff
@@ -101,13 +101,13 @@ See [API](#api) for all available options.
   area in your app scrollable based on viewport size, please use custom CSS
   with media queries instead.
 
-- ScrollView only supports **scrolling in single direction at a time.** It crops
-  content that could possibly overflow in the other direction because additional
-  scrollbars would be unreachable under scrolling shadows. If you need your
-  content to be ready for bi-directional scrolling, either consider using just
-  `overflow: auto` instead of ScrollView, or make your content scrollable before
-  putting it into ScrollView and make sure its scrollbars don't collide with
-  scrolling shadows.
+- ScrollView only supports **scrolling in a single direction at a time.** It
+  crops content that would possibly overflow in the other direction because
+  additional scrollbars would be unreachable under scrolling shadows. If you
+  need your content to be ready for bi-directional scrolling, either consider
+  using just `overflow: auto` instead of ScrollView, or make your content
```

Comment:
No, I'm not suggesting to put `ScrollView` into `ScrollView` in any possible way.

- _If you need your content to be ready for bi-directional scrolling,_
  - _either consider using just `overflow: auto` instead of ScrollView,_ — not here …
  - _or make your content scrollable before putting it into ScrollView …_ — and not here either.

Not to forget the very first part in bold:

> ScrollView only supports scrolling in single direction at a time.

As for now, there is no RUI component that allows or suggests nesting. I don't think it needs to be mentioned explicitely. It's way too creative thinking beyond my imagination…